kuma/payments/views.py

```diff
@@ -23,21 +31,33 @@ def index(request):
 
 
 @waffle_flag("subscription")
-@ensure_wiki_domain
 @never_cache
-def contribute(request):
-    return render(request, "payments/payments.html")
+def thank_you(request):
+    return render(request, "payments/thank-you.html")
+
+
+@waffle_flag("subscription")
+@never_cache
+@require_POST
+def send_feedback(request):
+    """
+    Sends feedback to Google Analytics. This is done on the
+    backend to ensure that all feedback is collected, even
+    from users with DNT or where GA is disabled.
+    """
+    data = json.loads(request.body)
```

Comment:
We should be consistent and do a check here too. At least something like this:
```python
feedback = (data.get('feedback') or '').strip()
if not feedback:
    return HttpResponseBadRequest("no 'feedback'")
```

Reply:
Added, thank you!